Themes/nightfall/nightfall.css

```diff
@@ -0,0 +1,2920 @@
+	padding-top: 4px;
+	padding-top: 2px;
+}
+
+.reblog_follow_button {
+	padding-top: 4px;
+}
+
+/*fix spacing on post header text to not cut off right of Follow button*/
+.post_full .post_info {
+	overflow: unset;
+}
+
+/*fix spacing on post header text (username + Follow) for Followed Tag posts*/
+.post_full.no_source .post_info_fence, 
+.post_full .post_action_follow {
+	padding-top: 2px;
+}
+
+.post_full .post_action_follow {
+	padding-top: 4px;
+}
+
+/*fix spacing on post header text (the tooltip for removing post) for Followed Tag posts*/
+.post_full .post_action_wrapper {
+	padding-top: 2px;
+}
+
+/*fix font size for Followed Tag posts*/
+.post_full .post_action_follow .follow_link {
+	font-size: 100%;
+}
+/*make Follow button look like an actual button done*/
+
+/*to make blog username part of Follow pill*/
+.post_full .post_info .post_info_fence.has_follow_button .reblog_source .post_info_link, 
+.dockable_video_embed .post_header .reblog_source .post_info_link {
+	background-color: #c4215d;
+	color: #DDD;
+	padding: 5px 5px 5px 10px;
+	border-radius: 300px 0px 0px 300px;
+	height: 25px;
+	border-rightOFF: hsla(338, 71%, 30%, 1) 2px solid;
+	display: inline-block;
+	margin-top: -2px;
+}
+
+.reblog_follow_button, 
+.worded-follow-button {
+	margin-left: 0px;
+}
+
+.reblog_follow_button .follow-text, 
+.worded-follow-button .follow-text, 
+.search-follow-button .follow-text, 
+.messaging-inbox .inbox-compose-toggle .compose-start {
+	border-radius: 0px 300px 300px 0px;
+	padding: 4px 10px 0px 4px;
+}
+/*to make blog username part of Follow pill done*/
+
+/*fixing the post header's spacing and stuff, bc Tumblr doesn't know how to do anything*/
+.post.post_full, 
+.peepr-body .post .post_wrapper {
+	padding: 0px;
+}
+
+.post.post_full .post_header, 
+.peepr-body .post.post_full .post_header, 
+.post .post.dockable_video_embed .post_header {
+	margin: 0px;
+	padding: 15px 20px;
+	margin-bottom: 0px;
+	line-height: unset;
+	line-height: 15px;
+	height: unset;
+	display: inline-block;
+	width: 100%;
+}
+
+.post.post_full .post_info {
+	vertical-alignOFF: middle;
+	margin: 0px;
+	padding: 0px;
+	max-width: 85%;
+}
+
+.post_full .post_info_fence.has_follow_button {
+	margin: 0;
+	padding: 0;
+	display: inline-block;
+	text-overflow: unset;
+	max-width: 100%;
+	vertical-align: middle;
+}
+
+.post_full .post_header .post_info .post_info_link:first-child, 
+.post .dockable_video_embed .post_header .post_info_tumblelog {
+	display: inline-block;
+	vertical-align: middle;
+}
+
+.post_full .post_info .post_info_fence .reblog_source,
+.post_full .post_info .post_info_fence.has_follow_button .reblog_source {
+	display: inline-block;
+	vertical-align: middle;
+}
+
+.post .post_info .reblog_icon, 
+.peepr-body .post .post_info .reblog_icon {
+	margin: 0px;
+	padding: 0px;
+	font-size: 0px;
+	vertical-align: middle;
+	display: inline;
+}
+
+.post .post_info .reblog_icon:before {
+	content: "\EA92";
+	top: -1px; /*bc icon itself is not centred in the font*/
```

Comment:
this is probably unnecessary, since visually centering icons always requires a pixel nudge or two (especially depending on surrounding context)